code/__DEFINES/colors.dm

```diff
@@ -11,6 +11,18 @@
 #define FIXED_COLOUR_PRIORITY 4
 ///how many colour priority levels there are.
 #define COLOUR_PRIORITY_AMOUNT 4
+/// If this is a plain atom color
+#define ATOM_COLOR_TYPE_NORMAL "normal"
+/// If this is a color filter
+#define ATOM_COLOR_TYPE_FILTER "filter"
+// Indexes for color arrays
+#define ATOM_COLOR_VALUE_INDEX 1
+#define ATOM_COLOR_TYPE_INDEX 2
+#define FILTER_ATOM_PRIORITY_COLOR "atom_priority_color"
```

Comment:
why is this named like a priority instead of like a filter name?

Reply:
should define the priority as well IMO, since it is reused. may be worth making -1 or someodd?